index.md

```diff
@@ -4,6 +4,16 @@
 
 {% include update.html %}
 
+ <h3 class="nhsuk-warning-callout__label">
+    Important<span class="nhsuk-u-visually-hidden">:</span>
+  </h3>
+  <p>This project is currently in development. For more information please contact the<a
```

Comment:
Thank you @maryamanuelnhsx. I added a relative link as it might be someone else who is deploying the template